### PR TITLE
Added Licence, contributions, ioshields, Changelog, Logo and version tag

### PR DESCRIPTION
**Pull request by rickstaa**
@ChanderG Here is the newly updated pull request. I now only uploaded the `.svgs` since these are scripted files instead of binaries. I further added the `auto-changelog` and `bumpversion` guide to the `contributions.md` file. I will implement the other features later this year. Let me know which features you like or dislike.

### CODE REVIEW

**Comment by rickstaa** [comment]
I forgot to create the `beginner` Projects that are a beginner-friendly label that I mentioned in the `contribution.md` document. If you can do this or give me the permissions to do this that would be great.

File: contributing.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+The auto-changelog tool by [@CookPete](https://github.com/CookPete/) is a small command line tool for generating a changelog from git tags and commit history. This tool can be used as follows:
+
+1.  Install NodeJS according to [the official NodeJS documentation](https://www.npmjs.com/get-npm).
+2.  Install the auto-changelog tool by running the `npm install -g auto-changelog` command.
+3.  Go to the main repositroy folder.
+4.  Commit any stages changes.
+5.  Bump the version using the bumpversion tool.
+6.  Run the `auto-changelog` command.
+7.  Commit the changes using `git commit -m "Updated CHANGELOG.md".
```
Comment (ChanderG) [inline contributing.md]: The inline code section ending is missed.

**Comment by ChanderG** [comment]
@rickstaa , looks good. One question before merging.

Ideally I would "Squash and merge". But, this means there will be a single commit at the end of the day. If this is ok with you, I'll merge this immediately; you can submit a follow up PR just to correct the changelog (as the original commits will not show up now).

Otherwise, if you would prefer to manage this as 2-4 commits, you can rebase interactively on your branch and push.

File: contributing.md (hunk 1 above)
Comment (rickstaa) [reply contributing.md]: You are right I missed that one! Good catch!

**Comment by rickstaa** [comment]
@ChanderG Squash and merge sounds very clean. Let's make that the default merging method. Thanks for bringing that to my attention! I wasn't aware of that method.

**Pull request merged**